fetch/api/redirect/redirect-origin.js

```diff
@@ -34,6 +34,7 @@ for (var code of [301, 302, 303, 307, 308]) {
     testOriginAfterRedirection("Same origin to same origin redirection " + code, redirectUrl, locationUrl, code, null);
     testOriginAfterRedirection("Same origin to other origin redirection " + code, redirectUrl, corsLocationUrl, code, get_host_info().HTTP_ORIGIN);
     testOriginAfterRedirection("Other origin to other origin redirection " + code, corsRedirectUrl, corsLocationUrl, code, get_host_info().HTTP_ORIGIN);
+    testOriginAfterRedirection("Other origin to same origin redirection " + code, corsRedirectUrl, locationUrl + "&cors", code, "null");
```

Comment:
This result isn't right - according to step 10 of [HTTP-redirect fetch](https://fetch.spec.whatwg.org/#concept-http-redirect-fetch), the post-redirection request should now have an opaque origin, and according to the [origin serialization algorithm](https://html.spec.whatwg.org/multipage/browsers.html#ascii-serialisation-of-an-origin) that means the request should contain an `Origin: null` header.